transcom/mymove
# B-18180 removed unused ppm fields from moves model

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A871874)

## Summary

Removed PPM specific fields in the moves model that aren't used.
Functionality of the app should not be effected.

### How to test

1. Login as a service member and create a ppm and hhg
2. Login as a service counselor and approve shipments
3. Everything should be working with no noticeable changes

### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
